docs/site/src/demos/bottom-navigation/LabelBottomNavigation.js

```diff
@@ -32,19 +36,19 @@ export default class LabelBottomNavigation extends Component {
         >
           <BottomNavigationButton
             label="Recents"
-            icon={<span className="material-icons">restore</span>}
+            icon={<RestoreIcon />}
```

Comment:
Oups, it's broken:
![capture d ecran 2017-01-01 a 13 17 14](https://cloud.githubusercontent.com/assets/3165635/21581157/b4c76334-d024-11e6-9100-df01cb13e14f.png)